Services/JobRunner/StartupHelper.cs

```diff
@@ -0,0 +1,32 @@
+﻿// *********************************************************************
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License
+// *********************************************************************
+
+using Microsoft.AspNetCore;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+
+namespace JobRunner
+{
+    /// <summary>
+    /// Helper that gets called from startup for the JobRunner service.
+    /// </summary>
+    public static class StartupHelper

```

Comment:
Can we reuse DataXServiceStartup ?

Reply:
DataXServiceStartup is very tightly coupled with DataX code.